x-pack/docs/en/rest-api/security/activate-user-profile.asciidoc

```diff
@@ -0,0 +1,10 @@
+[role="xpack"]
+[[security-api-activiate-user-profile]]
+=== Activate user profile API
+++++
+<titleabbrev>Activate user profile</titleabbrev>
+++++
+
+Creates or updates the user profile on behalf of another user
```

Comment:
I find the "on behalf" bit redundant.

Reply:
It's to emphasize that there are two users involved in this API, the caller and the target user for creating the profile document. I'll leave it as is and we can come back to it in #83495